packages/urban-bot/src/node.ts

```diff
@@ -52,13 +48,16 @@ export function createNode<Bot extends UrbanBot>(
         ...node,
         childNodes: [],
         data,
-    } as UrbanNode<Bot>;
+    } as UrbanNode<Metadata>;
 }
 
-export function appendChildNode<Bot extends UrbanBot>(
-    parentNode: UrbanNode<Bot> | UrbanNodeRoot,
-    childNode: UrbanNode<Bot>,
+export function appendChildNode<Metadata extends UrbanBotMeta>(
+    parentNode: UrbanNode<Metadata> | UrbanNodeRoot,
+    childNode: UrbanNode<Metadata>,
 ) {
+    // TODO: Fix me UrbanNodeRoot.
+    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+    // @ts-ignore
```

Comment:
**childNodes** is
``` ts
UrbanNode<Metadata>
``` 
but  **parentNode** is  
 ``` ts
 UrbanNode<Metadata> | UrbanNodeRoot
```

Reply:
// FIXME UrbanNodeRoot
I'll take a look

Reply:
Resolve